### PR TITLE
use Kb.Text for text filepreview on desktop; fix ts mime

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: shared/fs/filepreview/text-view.desktop.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import * as Styles from '../../styles'
+import * as Kb from '../../common-adapters'
+import * as SafeElectron from '../../util/safe-electron.desktop'
+import {Props} from './text-view'
+
+const TextView = (props: Props) => {
+  const {url} = props
+  const [content, setContent] = React.useState('')
+  React.useEffect(() => {
+    const req = SafeElectron.getRemote().net.request({method: 'GET', url})
```
Comment (chrisnojima) [inline shared/fs/filepreview/text-view.desktop.tsx]: lets chat about this approach

**Comment by chrisnojima** [comment]
@keybase/react-hackers this fixes text in files tab

File: shared/fs/filepreview/text-view.desktop.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      try {
+        if (req.readyState === XMLHttpRequest.DONE && req.status === 200) {
+          setContent(req.responseText)
+        }
+      } catch (e) {}
+    }
+    try {
+      req.open('GET', url)
+      req.send()
+    } catch (e) {}
```
Comment (jakob223) [inline shared/fs/filepreview/text-view.desktop.tsx]: Do we want to show the user a "Could not load file" or something in this case?

**Comment by jakob223** [comment]
(otherwise LGTM)

**Comment by chrisnojima** [comment]
We can add that later if it really happens

**Pull request merged**